tests/test_relative_free_energy.py

```diff
@@ -448,6 +450,29 @@ def test_rbfe_fallback_from_near_zero_overlap():
         )
 
 
+@patch("timemachine.fe.rbfe.compute_u_kn")
+def test_rebalance_lambda_schedule(mock_compute_u_kn):
+    # has some issues with default PyMBAR settings
+    with path_to_internal_file("timemachine.testsystems.data", "u_kn_unstable.npz") as path_to_npz:
+        npz = np.load(path_to_npz)
+        u_kn, N_k, initial_lambdas = npz["u_kn"], npz["N_k"], npz["initial_lambdas"]
+
+    mock_compute_u_kn.return_value = (u_kn, N_k)
+
+    final_lambdas = []
+
+    def initial_state_fxn(lamb: float):
+        final_lambdas.append(lamb)
+        return
+
+    initial_states = [Mock(lamb=lamb) for lamb in initial_lambdas]
+    trajectories = [None for _ in range(len(initial_states))]
+    rebalance_lambda_schedule(initial_states, initial_state_fxn, trajectories, 2 / 3)
```

Comment:
nit/optional: seems slightly clearer to me to do (untested):
```suggestion
    def initial_state_fxn(lamb: float):
        return lamb

    initial_states = [Mock(lamb=lamb) for lamb in initial_lambdas]
    trajectories = [None for _ in range(len(initial_states))]
    final_lambdas = rebalance_lambda_schedule(initial_states, initial_state_fxn, trajectories, 2 / 3)
```

Reply:
updated in https://github.com/proteneer/timemachine/pull/1516/commits/b03f0d3f6c70cd53511cbcc982ea04bf482f7e8b